scripts/o3de/o3de/manifest.py

```diff
@@ -335,6 +334,28 @@ def get_engine_templates() -> list:
                         engine_object['templates'])) if 'templates' in engine_object else []
     return []
 
+def is_sdk_engine(engine_name:str = None, engine_path:str or pathlib.Path = None) -> bool:
+    """
+    Queries if the engine is an SDK engine that was created via the CMake install command
+    :param engine_name: Name of a registered engine to lookup in the ~/.o3de/o3de_manifest.json file
+    :param engine_path: The path to the engine
+    :return True if the engine.json contains an "sdk_engine" field which is True
+    """
+    engine_json_data = get_engine_json_data(engine_name, engine_path)
+    if engine_json_data:
+        return engine_json_data.get('sdk_engine', False)
+
+    if engine_path:
+        logger.error('Failed to retrieve engine json data for the engine at '
```

Comment:
`errors` are for situations were the process stops. If the process is continuing with 'Treating engine as source engine.', then this should be a warning instead.